base/redhat-8/Dockerfile

```diff
@@ -12,9 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# NOTE that since OpenShift Container Platform 3.11
+# the container catalog moved from registry.access.redhat.com to registry.redhat.io
+# So at some point before they deprecate the old registry we have to make sure that
+# we have access to the new registry and change where we pull the ubi image from.
 FROM registry.access.redhat.com/ubi8/ubi-minimal
-LABEL maintainer="support@splunk.com"
+LABEL name="splunk" \
+      maintainer="support@splunk.com" \
+      vendor="splunk" \
+      version="7.3.1" \
```

Comment:
This is fine to hardcode for now, but in the future we should update this to be whatever the splunk version is